Project.toml

```diff
@@ -35,13 +35,14 @@ Tracker = "9f7883ad-71c0-57eb-9f7f-b5c9e6d3789c"
 AbstractMCMC = "0.5.2"
 AdvancedHMC = "0.2.20"
 AdvancedMH = "0.4"
-Bijectors = "0.4.0, 0.5, 0.6"
+Bijectors = "0.6.2"
 BinaryProvider = "0.5.6"
-Distributions = "0.21.11, 0.22"
-DistributionsAD = "0.4.1"
+Distributions = "0.22"
+DistributionsAD = "0.4.3"
 DocStringExtensions = "0.8"
 DynamicPPL = "0.4"
 EllipticalSliceSampling = "0.2"
+FiniteDifferences = "0.9"
```

Comment:
Do we need the compatibility bound on the test dependency `FiniteDifferences`? IIRC, by default, Pkg will remove it if you run `] up` and CompatHelper won't open PRs for updating it.

Reply:
Removed